src/transforms/reduce/mod.rs

```diff
@@ -77,12 +77,11 @@ impl ReduceState {
     fn new(e: LogEvent, strategies: &IndexMap<String, MergeStrategy>) -> Self {
         Self {
             stale_since: Instant::now(),
-            // TODO: all_fields alternative that consumes
```

Comment:
![](https://media.giphy.com/media/cJzkUNfmjeBM7q2gBb/giphy.gif)

As it was foretold.